[DPMMA-2108] Add UTM to Asset download #12098 (#157)
mautic/user-documentation: docs/components/assets.rst

.. warning:: 
    Once deleted, you can't retrieve an Asset, and statistics relating to the number of downloads for that Asset are no longer be available. Contact Points accumulated as a result of accessing the resource remain. It's recommended where possible to unpublish Assets which are no longer in use - in future there may be an archive feature.
    

Add UTM to Asset
================

> You can add a resource of UTM parameters to the download link. We can use UTM data in the resource download report.

Reviewer: ```suggestion
UTM parameters appended to the download link means that UTM data is available in the resource download Report.
```

Fix the style - Vale is flagging a warning due to the use of '[We](https://github.com/mautic/user-documentation/actions/runs/4891096079/jobs/8731311211?pr=157#step:4:30)' and the lack of capitalisation of [Report](https://github.com/mautic/user-documentation/actions/runs/4891096079/jobs/8731311211?pr=157#step:4:38).

I actually think that this fits better in the Creating Assets section, as that's when the user is most likely to need to have this information - what do you think?
